tests/test_tmfa_tools.py

```diff
@@ -0,0 +1,37 @@
+# these test is to ensure that the object is not None, has non-zero lengths and that not all the values are 0
+def test_get_fluxes(create_small_model_for_tesing):
+    from AutoFlow_OmicsDataHandling.tmfa_tools import get_fluxes
+
+    calc = get_fluxes(create_small_model_for_tesing)
+    assert (
```

Comment:
Do the values change between calls to `get_fluxes`?  If not, I would recommend adding in tests for absolute values or at least value ranges to compliment the tests that you have here.